src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -128,6 +128,11 @@ public string HubName
         /// </value>
         public int ExtendedSessionIdleTimeoutInSeconds { get; set; } = 30;
 
+        /// <summary>
+        ///  States that will override an existing orchestrator when attempting to start a new orchestrator with the same instance Id.
+        /// </summary>
+        public OverridableStates OverridableExistingInstanceStates { get; set; } = OverridableStates.AnyState;
```

Comment:
The default should actually be `OverridableStates.IfTerminatedFailedOrCompleted`.